keyboards/nk65/nk65.c

```diff
@@ -13,6 +13,42 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
- #ifndef RGB_BACKLIGHT_NK65
- #error RGB_BACKLIGHT_NK65 not defined, recheck config.h
- #endif
+#ifndef RGB_BACKLIGHT_NK65
+#error RGB_BACKLIGHT_NK65 not defined, recheck config.h
+#endif
+
+#include "nk65.h"
+#include "drivers/issi/is31fl3733.h"
+
+/* Indicator LEDS are part of the LED driver 
+ * Top LED is blue only. LED driver 2 RGB 7 Green channel
+ * Middle LED is blue and red. LED driver 2 RGB 6 Red and Blue channel
+ * Bottom LED is red only LED driver 2 RGB 6 Green channel.
+ */
+__attribute__((weak)) bool led_update_kb(led_t led_state) {
```

Comment:
```suggestion
bool led_update_kb(led_t led_state) {
```

This doesn't need to be weak, as the result of `led_update_user()` determines whether the code here will run.